package.json

```diff
@@ -49,8 +49,10 @@
   },
   "prettier": "tomer/prettier",
   "dependencies": {
+    "@csstools/css-parser-algorithms": "^2.1.1",
+    "@csstools/css-tokenizer": "^2.1.1",
+    "fontkit": "^2.0.2",
```

Comment:
This is a peer dependency in `fontpie-calc`, so must be a dependency here for there package to work.

Reply:
Makes sense